pxm-manifest-spec.md

```diff
@@ -24,7 +24,7 @@ We will discuss each below:
 
 ### 3.1.  sources
 
-The `sources` object MUST be a list of source images to be rendered. Each element must be an S3 URI to a valid raster dataset, one of the following:
+The `sources` object MUST be a list of source images to be rendered. Each element must be an unique S3 URI to a valid raster dataset, one of the following:
```

Comment:
@vincentsarago one thing I learned from writing specs for the IETF is that less "shouting" is better :) Let's replace "object MUST be a list" with "object is a list". The second "must" really does need to be a "MUST" in my opinion, because our system will fail if this isn't followed.